src/apphosting/index.ts

```diff
@@ -147,7 +178,13 @@ export async function doSetup(
       },
     },
   });
-  createRolloutSpinner.succeed(`Your backend is now deployed at:\n\thttps://${backend.uri}`);
+  createRolloutSpinner.succeed("Rollout complete");
+  if (!(await tlsReady(url))) {
+    const tlsSpinner = ora("Waiting for TLS certificate");
```

Comment:
you probably want tlsSpinner = ora(...).start()

Reply:
Whoops! Maybe Ora isn't as easy to use as I thought!